imap_processing/tests/ultra/unit/test_ultra_l1c_pset_bins.py

```diff
@@ -197,20 +197,6 @@ def test_get_spacecraft_sensitivity():
         df_sensitivity_test.to_numpy(), expected_sensitivity.to_numpy(), atol=1e-6
     )
 
-
-@pytest.mark.external_test_data
-def test_grid_sensitivity():
```

Comment:
Do you not need this test anymore?

Was it only this one test causing all the failures? :( Really weird, hopefully it wasn't just luck for this single run passing!